### ISSUE TITLE
Drop dependencies

### ISSUE THREAD

**Issue by brunolemos** [bmcmahen/react-image-enlarger#2]
I see you added `react-spring` as a dependency, but that's usually an app dependency, not a library dependency. Maybe the library should be dependency free?

https://bundlephobia.com/result?p=react-spring@9.0.0-beta.31

**Comment by bmcmahen** [comment]
Yeah, I'm never really sure if react-spring should be considered a peer dependency or not. It's required either way... and this sort of lib is really difficult to build if you're not using some sort of spring-based animation. Can you explain the benefits of making it a peer dependency instead?
